components/match2/wikis/starcraft/match_legacy.lua

```diff
@@ -105,7 +105,9 @@ function p.storeGames(match, match2)
 			submatch.winner = game.winner or ''
 			submatch.walkover = game.walkover or ''
 			submatch.finished = match2.finished or '0'
-			submatch.resulttype = game.resulttype
+			if submatch.resulttype ~= 'submatch' then
```

Comment:
this is always true as the left side is always nil
```suggestion
			if game.resulttype ~= 'submatch' then
```

Reply:
oh im stupid